py/embedded-server/README_PyPi.md

```diff
@@ -0,0 +1,27 @@
+# deephaven-server
+
+The Dockerless version of [deephaven-core](https://www.github.com/deephaven/deephaven-core).
+
+Deephaven Community Core is a real-time, time-series, column-oriented analytics engine with relational database features. It is an open version of [Deephaven Enterprise](https://deephaven.io/enterprise/), which serves as the data backbone for prominent hedge funds, banks, and financial exchanges.
+
+`deephaven-server` enables users to run Deephaven Community Core without Docker.
```

Comment:
I'm not a fan of explicitly calling our main product a "docker" product; and thus, `deephaven-server` is "Deephaven minus docker". I think highlighting that this is a python-native way of starting Deephaven is what we should strive for conveying.